test/match-base.js

```diff
@@ -11,7 +11,7 @@ var expect = [
 ]
 
 if (process.platform !== 'win32')
-  expect.push('a/symlink/a', 'a/symlink/a/b/c/a')
+  expect.push('a/symlink/a')
```

Comment:
This is a breaking change, and a deviation from Bash 4.3.

```
$ cd test/fixtures/

$ echo **/a*
a a/abcdef a/abcfed a/symlink/a a/symlink/a/b/c/a

$ echo $BASH_VERSION
4.3.30(6)-release
```

The `**` should be followed through to `a/symlink/a/b/c/a`, but then it should refuse to go any further subdirectories *past* the symlink.